src/main/java/org/mifos/connector/ams/zeebe/ZeebeUtil.java

```diff
@@ -1,19 +1,39 @@
 package org.mifos.connector.ams.zeebe;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import org.apache.camel.Exchange;
-import org.json.JSONObject;
+import org.mifos.connector.common.ams.dto.LoanRepaymentDTO;
+import org.mifos.connector.common.channel.dto.TransactionChannelRequestDTO;
+import org.mifos.connector.common.gsma.dto.CustomData;
+import org.mifos.connector.common.gsma.dto.GsmaTransfer;
+import org.mifos.connector.common.mojaloop.dto.*;
+import org.mifos.connector.common.mojaloop.type.*;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
 
-import java.util.LinkedHashMap;
+import java.math.BigDecimal;
+import java.text.Format;
+import java.text.SimpleDateFormat;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+import java.util.List;
 import java.util.Map;
 
+import static org.mifos.connector.ams.zeebe.ZeebeVariables.ACCOUNT_IDENTIFIER;
+import static org.mifos.connector.ams.zeebe.ZeebeVariables.ACCOUNT_NUMBER;
 
+@Component
 public class ZeebeUtil {
 
+
     private static Logger logger = LoggerFactory.getLogger(ZeebeUtil.class);
     private static ObjectMapper objectMapper = new ObjectMapper();
+    @Value("#{'${accountPrefixes}'.split(',')}")
```

Comment:
if we consider the accountPrefixes in the yaml as an array, then we don't need to split it again. use array everywhere.

Reply:
This was suggested by Danish as it was being used previously.

Reply:
We should move towards better solution.